werewolves/lib/main.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:flutter/material.dart';
-import 'package:werewolves/ui/pages/start_page.dart';
+import 'package:werewolves/ui/pages/start/main_menu.dart';
```

Comment:
so the intention for the folder `pages` is to hold widgets that represent a whole page (login, landing page(homePage), aboutUs ... etc)

smaller widgets can be moved to `ui/widgets` and can be grouped in more folders (or not)

Reply:
Is this in reference to the main_menu_button_list.dart.  I like that distinction but I don't think I've made many general widgets that should go into ui/widgets yet.  I think my code could be refactored (and probably should, as I have no experience building large app infrastructure) so let me know how we want to do that